test/tests_for_sdkmr/ee2_SDKMethodRunner_test_EE2Runjob_test.py

```diff
@@ -237,6 +237,43 @@ def test_run_job(self, rq_mock, condor_mock):
         job_id = runner.run_job(params=job)
         print(f"Job id is {job_id} ")
 
+    @requests_mock.Mocker()
+    @patch("lib.execution_engine2.utils.Condor.Condor", autospec=True)
+    def test_run_job_batch(self, rq_mock, condor_mock):
+        rq_mock.add_matcher(
+            run_job_adapter(
+                ws_perms_info={"user_id": self.user_id, "ws_perms": {self.ws_id: "a"}}
+            )
+        )
+        runner = self.getRunner()
+        runner.get_condor = MagicMock(return_value=condor_mock)
+        job = get_example_job(user=self.user_id, wsid=self.ws_id).to_mongo().to_dict()
+        job["method"] = job["job_input"]["app_id"]
+        job["app_id"] = job["job_input"]["app_id"]
+        job["service_ver"] = job["job_input"]["service_ver"]
```

Comment:
Is it worth creating three separate job data objects here so you can check them individually, rather than three copies of the same job?

Reply:
I am interested more so in whether the jobs are run and there are child jobs, I don't think the contents of the jobs matter.